### PR TITLE
[Opta] fix deflected pass end coordinates and result

### PR DESCRIPTION
**Pull request by DriesDeprest**
Fix for issue described in: https://github.com/PySport/kloppy/issues/274

### CODE REVIEW

**Comment by JanVanHaaren** [comment]
@probberechts Could you please also have a look at this pull request? Thank you!

File: kloppy/infra/serializers/event/opta/deserializer.py [hunk 1]
```diff
@@ -287,6 +298,22 @@ def _parse_pass(raw_qualifiers: Dict[int, str], outcome: int) -> Dict:
 
     qualifiers = pass_qualifiers + overall_qualifiers
 
+    if next_event is not None and next_next_event is not None:
```
[the remark this answers is not in the document]
Comment (koenvo) [reply kloppy/infra/serializers/event/opta/deserializer.py:301]: @DriesDeprest the comment looks like a nice addition

**Comment by DriesDeprest** [comment]
@koenvo added the comment, but checks seems to be failing due to unrelated issue: https://github.com/PySport/kloppy/actions/runs/9064091328/job/24901521491

**Comment by koenvo** [comment]
Can you please merg master in to make sure the tests run

**Comment by DriesDeprest** [comment]
A painful merge conflict to resolve due to the refactoring of the statsperform parser 😅 , but this one should be ready to merge as well @koenvo!

**Comment by koenvo** [comment]
Oh no. Again merge conflicts :-(

**Comment by DriesDeprest** [comment]
resolved them

**Pull request merged**